appengine/src/main/java/com/google/googleinterns/gscribe/config/MySQLConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.google.googleinterns.gscribe.config;
+
+public class MySQLConfig {
+  private String url;
+  private String userName;
+  private String password;
+
+  public MySQLConfig(String url, String userName, String password) {
```

Comment:
Once the url and credentials have been moved to properties files, you can inject a PropertiesParser here and extract the values from it and set them.